Tasks/SshV0/ssh2helpers.ts

```diff
@@ -63,23 +63,33 @@ function handleStreamClose(command: string, stdErrWritten: boolean, defer: Q.Def
 }
 
 /**
- * Uses scp2 to copy a file to remote machine
- * @param scriptFile
- * @param scpConfig
+ * Uses sftp to copy a file to remote machine
+ * @param src
+ * @param dest
+ * @param sftpConfig
  * @returns {Promise<string>}
  */
-export function copyScriptToRemoteMachine(scriptFile: string, scpConfig: any): Q.Promise<string> {
+export async function copyScriptToRemoteMachine(src: string, dest: string, sftpConfig: any): Promise<string> {
```

Comment:
Could you please define some interface for sftpConfig? Or maybe there some types we could import, could we use https://www.npmjs.com/package/@types/ssh2-sftp-client?